config.json

```diff
@@ -304,6 +305,7 @@
     "pdp-sale-price-label": "Now:",
     "pdp-non-sale-price-label": "Was:",
     "pdp-retail-price-label": "MSRP:",
+    "pdp-custom-fields-tab-label": "Specifications",
```

Comment:
Can we also change the label to be more generic like "Additional Information" ? 🍹